tests/integration/nmstatectl_test.py

```diff
@@ -206,6 +208,19 @@ def test_automatic_rollback(eth1_up):
         assertlib.assert_state(clean_state)
 
 
+def test_version_argument():
+    cmds = ["nmstatectl", "--version"]
+    ret = cmdlib.exec_cmd(cmds)
+    rc, out, err = ret
+    assert rc == cmdlib.RC_SUCCESS, cmdlib.format_exec_cmd_result(ret)
+    assert out.rstrip() == __version__
+
+def test_version_command():
+    ret = cmdlib.exec_cmd(VERSION_CMD)
+    rc, out, err = ret
```

Comment:
The `err` is not used, please change to:

`rc, out, _ = ret`

Reply:
Done.